mysos/scheduler/http.py

```diff
@@ -24,15 +24,17 @@ def __init__(self, scheduler, asset_dir):
   def create(self, clustername):
     """Create a db cluster."""
     cluster_name = clustername  # For naming consistency.
-    num_nodes = bottle.request.forms.get('num_nodes', default=3)
+    num_nodes = bottle.request.forms.get('num_nodes', default=1)
     cluster_user = bottle.request.forms.get('cluster_user', default=None)
     backup_id = bottle.request.forms.get('backup_id', default=None)
+    size = bottle.request.forms.get('size', default=None)
```

Comment:
there's no flag for this but just constant defaults - we can add flags for the defaults later when we add T-shirt sizing for #14. The logic that handles 'None' is in scheduler.py because I want to consolidate the validation code there and keep http.py simple.